pkg/remote/aws/vpc_security_group_supplier.go

```diff
@@ -16,59 +16,102 @@ import (
+	client                           ec2iface.EC2API
+	defaultSecurityGroupRunner       *terraform.ParallelResourceReader
+	securityGroupRunner              *terraform.ParallelResourceReader
 }
 
 func NewVPCSecurityGroupSupplier(runner *pkg.ParallelRunner, client ec2iface.EC2API) *VPCSecurityGroupSupplier {
-	return &VPCSecurityGroupSupplier{terraform.Provider(terraform.AWS), awsdeserializer.NewVPCSecurityGroupDeserializer(), client, terraform.NewParallelResourceReader(runner)}
+	return &VPCSecurityGroupSupplier{
+		terraform.Provider(terraform.AWS),
+		awsdeserializer.NewDefaultSecurityGroupDeserializer(),
+		awsdeserializer.NewVPCSecurityGroupDeserializer(),
+		client,
+		terraform.NewParallelResourceReader(runner.SubRunner()),
+		terraform.NewParallelResourceReader(runner.SubRunner()),
+	}
 }
 
 func (s VPCSecurityGroupSupplier) Resources() ([]resource.Resource, error) {
-	securityGroups, err := listSecurityGroups(s.client)
+	securityGroups, defaultSecurityGroups, err := listSecurityGroups(s.client)
 	if err != nil {
 		return nil, err
 	}
-	results := make([]cty.Value, 0)
-	if len(securityGroups) > 0 {
-		for _, securityGroup := range securityGroups {
-			sg := *securityGroup
-			s.runner.Run(func() (cty.Value, error) {
-				return s.readSecurityGroup(sg)
-			})
-		}
-		results, err = s.runner.Wait()
-		if err != nil {
-			return nil, err
-		}
+
+	for _, item := range securityGroups {
+		res := *item
+		s.securityGroupRunner.Run(func() (cty.Value, error) {
+			return s.readSecurityGroup(res)
+		})
+	}
+	securityGroupResources, err := s.securityGroupRunner.Wait()
+	if err != nil {
+		return nil, err
+	}
+
+	for _, item := range defaultSecurityGroups {
+		res := *item
+		s.defaultSecurityGroupRunner.Run(func() (cty.Value, error) {
+			return s.readSecurityGroup(res)
+		})
+	}
+	defaultSecurityGroupResources, err := s.defaultSecurityGroupRunner.Wait()
+	if err != nil {
+		return nil, err
 	}
-	return s.deserializer.Deserialize(results)
+
+	// Deserialize
+	deserializedDefaultSecurityGroups, err := s.defaultSecurityGroupDeserializer.Deserialize(defaultSecurityGroupResources)
+	if err != nil {
+		return nil, err
+	}
+	deserializedSecurityGroups, err := s.securityGroupDeserializer.Deserialize(securityGroupResources)
+	if err != nil {
+		return nil, err
+	}
+
+	resources := make([]resource.Resource, 0, len(securityGroupResources)+len(defaultSecurityGroupResources))
+	resources = append(resources, deserializedDefaultSecurityGroups...)
+	resources = append(resources, deserializedSecurityGroups...)
+
+	return resources, nil
 }
 
 func (s VPCSecurityGroupSupplier) readSecurityGroup(securityGroup ec2.SecurityGroup) (cty.Value, error) {
-	id := aws.StringValue(securityGroup.GroupId)
-	resSg, err := s.reader.ReadResource(terraform.ReadResourceArgs{
-		Ty: resourceaws.AwsSecurityGroupResourceType,
-		ID: id,
+	var Ty resource.ResourceType = resourceaws.AwsSecurityGroupResourceType
+	if securityGroup.GroupName != nil && *securityGroup.GroupName == "default" {
```

Comment:
It could be better to have this logic once, could you make an unexported method in this supplier and reuse it there and in `listSecurityGroups`, or pass a `isDefault` bool to this method when looping on resources.